packages/browser/core/src/core/auth/internal/StitchAuthImpl.ts

```diff
@@ -77,6 +110,98 @@ export default class StitchAuthImpl extends CoreStitchAuth<StitchUser>
     return super.loginWithCredentialInternal(credential);
   }
 
+  public loginWithRedirect(credential: StitchRedirectCredential): void {
+    const { redirectUrl, state } = prepareRedirect(
+      this,
+      credential,
+      RedirectTypes.Login
+    );
+
+    window.location.replace(
+      this.browserAuthRoutes.getAuthProviderRedirectRoute(
+        credential,
+        redirectUrl,
+        state,
+        this.deviceInfo
+      )
+    );
+  }
+
+  public hasRedirect(): boolean {
```

Comment:
[remove] we don't need this. A client should be redirecting themselves to a page/fragment for redirects

Reply:
You don't think it's useful as a convenience method?

Reply:
I don't. If we wanted this anyway we would need for this to be checking the url, not the storage.

Reply:
I would disagree with you @edaniels . by default, the login/link with redirect redirects to the page from which it was called, so if we don't have this method, I feel like we should get rid of that default behavior and force people to pick a redirect destination as part of the login/link method.

Reply:
👍